stock_picking_line_sequence/readme/CONTRIBUTORS.rst

```diff
@@ -1,4 +1,5 @@
 
+* Sulivan Lominchar <slominchar@archeti.com>
```

Comment:
Can you move sulivan to the end of the list?

Reply:
done